src/components/SelectedSurveyPage.jsx

```diff
@@ -26,6 +26,19 @@ export default class SelectedSurveyPage extends PureComponent {
     fetchSurvey(id);
   }
 
+  // TODO(ayushbhagat): Check for cycles.
+  getValidNextQuestionOptions = (id) => {
+    const { survey } = this.props;
+    const questions = survey.getIn(['body', 'questions']);
+    const options = [];
+    for (let i = 0; i < questions.size; i++) {
```

Comment:
Can we change `i` to `questionID` or something more descriptive ?
I know its pretty obvious from the code, but still good to be explicit.